components/log-viewer-webui/server/src/app.js

```diff
@@ -30,7 +44,36 @@ const app = async (clientDir, fastifyOptions = {}) => {
             root: clientDir,
         });
     }
+
+    await server.register(fastifyStatic, {
+        prefix: "/ir",
+        root: irDataDir,
+    });
+    await server.register(fastifyStatic, {
+        prefix: "/log-viewer",
+        root: logViewerDir,
+        decorateReply: false,
```

Comment:
Needed for any second and after registration of fastifyStatic: https://github.com/fastify/fastify-static#disabling-reply-decorator